scripts/functions.sh

```diff
@@ -58,6 +58,9 @@ export HOST_RCBOPS_REPO=${HOST_RCBOPS_REPO:-"http://rpc-repo.rackspace.com"}
 # Derive the rpc_release version from the group vars
 export RPC_RELEASE="$(/opt/rpc-openstack/scripts/artifacts-building/derive-artifact-version.sh)"
 
+# Upgrade of Elasticsearch
+export UPGRADE_ELASTICSEARCH=${UPGRADE_ELASTICSEARCH:-"no"}
```

Comment:
what is this and why is this here?

Reply:
By default deployment, the upgrade flag is supposed to be negative. My intending is wishing user to manually export this flag to true.

Reply:
doc this